src/wmnds/components/content-tiles/page-contents/page-contents.njk

```diff
@@ -0,0 +1,21 @@
+{% macro wmndsContentTilePageContents(params) %}
+
+<div class="wmnds-content-tile">
+    <div>
+    <h2 class="wmnds-content-tile__title--content">Page Contents</h2>
+    </div>
+    <div class="wmnds-content-tile--copy">
+    <div class="overview">
+    <p>Overview</p>
+    </div>
+    <a href="#" class="wmnds-col-1 wmnds-content-tile--copy__links">Information for drivers</a>
+    <a href="#" class="wmnds-col-1 wmnds-content-tile--copy__links">Information for bus passengers</a>
+    <a href="#" class="wmnds-col-1 wmnds-content-tile--copy__links">Information for tram passengers</a>
```

Comment:
Remove references to the grid. As grid styling shouldn't be used within a component as it can conflict with the end developers grid settings.
```suggestion
    <a href="#" class="wmnds-content-tile--copy__links">Information for tram passengers</a>
```